README.md

```diff
@@ -6,26 +6,48 @@ Host: Jiayu Peng (jiayupeng@), Xichen Huang (huangxichen@, co-host)
 
 ## Overview
 
-This repository includes code for
-
-* Evaluation of the "shuffling+flipping" algorithm for bloom filter (BF). In this algorithm, the "shuffling" refers to a random permutation of bits in the BF, and the "flipping" refers to independent flipping (with a certain probability) of bits in the BF. We evaluate this algorithm in terms of its differential privacy (and accuracy). 
+This repository includes code for evaluation of the "shuffling+flipping" algorithm for bloom filter (BF). 
+In this algorithm, the "shuffling" refers to a random permutation of bits in the BF, and the "flipping" refers to independent flipping (with a certain probability) of bits in the BF. 
+We evaluate this algorithm in terms of its differential privacy (and accuracy). 
 
 ### Quickstart
-It is recommended to use a virtual environment with this project. If you already
-have one, you can skip to the next step.
+It is recommended to use a virtual environment with this project. 
+If you already have one, you can skip to the next step.
 The quickest way to set up a virtual environment is by running:
-```
+```bash
 python3 -m venv env
 source env/bin/activate
 ```
 To install the requirements as well as the evaluation framework, simply run:
-```
+```bash
 pip install -r requirements.txt
 python setup.py install
 ```
-After these steps, the code and its dependencies will be installed as Python packages.
+After these steps, the code and its dependencies will be installed as Python 
+packages.
 
 ### Example
+`buffling` stands for "bit shuffling." 
 ```python
-import buffling
-```
\ No newline at end of file
+import buffling.privacy as bp
+import buffling.signal_to_noise_ratio as snr
+```
```

Comment:
And then how to run it? Seems that the "Example" section is not completed.

Reply:
I have more examples in the example/ folder. will add a pointer to those in the README.